### ISSUE TITLE
FEATURE REQUEST- ADD ITEM TAGS SUPPORT FOR BLEEDING CONFIG

### ISSUE THREAD

**Issue by Thelegendking** [Majrusz/MajruszsProgressiveDifficultyMod#253]
**REASON** - I would like to add many of my custom weapons for bleeding effect in the config but they have their own names, And they don't always end with "_sword" "_dagger" etc. It takes way too long to add them one by one on that list.

**SOLUTION** - So unless there is a way to add support for all the modded weapons with their different names at once that I'm not aware of this is the idea instead of adding weapons through their last name add another second option to add item tags like _"tags: "simplyswords:swords""_ (it might just say "swords" but trust me it doesn't just include swords) this way all the weapons get the bleeding effect those items which have this tag.

**RESULT** - This feature would make adding weapons on config super easy and super fast, it would especially be useful for big weapon mods like simplyswords which adds their alot of weapons with different custom names that doesn't really end with "_sword" or any other name.

**Comment by inmourning** [comment]
You can use regex
For example in my modpack:

``` "id": "{regex}simplyswords:(soulkeeper|twisted_blade|soulstealer|soulrender|soulpyre|shadowsting|whisperwind|enigma|magiblade|magispear|bramblethorn|frostfall|icewhisper|livyatan|sunfire|hiveheart|stars_edge|tempest|flamewind|emberlash|wickpiercer|waxweaver|harbinger|ribboncleaver|slumbering_lichblade|waking_lichblade|awakened_lichblade|dormant_relic|righteous_relic|tainted_relic|decaying_relic|magiscythe)", ```

**Comment by Thelegendking** [comment]
> You can use regex For example in my modpack:
> 
> `"id": "{regex}simplyswords:(soulkeeper|twisted_blade|soulstealer|soulrender|soulpyre|shadowsting|whisperwind|enigma|magiblade|magispear|bramblethorn|frostfall|icewhisper|livyatan|sunfire|hiveheart|stars_edge|tempest|flamewind|emberlash|wickpiercer|waxweaver|harbinger|ribboncleaver|slumbering_lichblade|waking_lichblade|awakened_lichblade|dormant_relic|righteous_relic|tainted_relic|decaying_relic|magiscythe)",`

Yeah already did that since there's no other option, that's why I made this suggestion post. Current process of adding custom weapons through "regex" takes too much time, by being able to just put weapon tag which is used by any mod makes it compatible very simple and very fast in just one or two or three words and done now those weapons have bleeding effect too.
